packages/terra-consumer-nav/src/components/quick-links/QuickLink.scss

```diff
@@ -0,0 +1,32 @@
+@import '../../variables';
+
+:local {
+  .quicklinks-container {
+    margin: 0 25px;
+  }
+
+  .quicklink {
+    background: var(--terra-consumer--quicklink-background, #fff);
+    border: 0;
+    border-radius: 3px;
+    color: var(--terra-consumer--quicklink-color, #3d3d3d);
+    display: inline-block;
+    font-family: inherit;
+    font-size: 16px;
+    line-height: 35px;
+    margin-bottom: 10px;
+    padding: 0;
+    text-align: center;
+    text-decoration: none;
+    width: 100%;
+
+    &:hover {
+      background: var(--terra-consumer--quicklink-background-hover, #eee);
+      cursor: pointer;
+    }
```

Comment:
there are no callouts or reqs as of now for hover/focus, I added this as I feel it's needed and is easily changeable/theme-able for clients